### ISSUE TITLE
🆕 Software Suggestion | 

### ISSUE THREAD

**Issue by simooooone** [privacytoolsIO/privacytools.io#768]
## Basic Information

**Name:**
Berryer

**Category:**
Domain Name System (DNS)

**URL:**
https://github.com/riminilug/berryer

## Description
Procedure to install, configure and update a working suite of privacy respectful docker containers and docker itself on a Raspberry PI 3B+. Other platforms has not been tested as I am almost alone at the moment.

**Comment by blacklight447-ptio** [comment]
so berryer is not really a solution on its own, but more of a way to easily roll out existing solutions?

**Comment by simooooone** [comment]
Exactly, at the moment it is a script to install docker and privacy services such as OpenVPN and Pi-Hole on top of an existing Raspbian installation
